app/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,146 @@
+    <string name="session_details_section_title_links">リンク</string>
+    <string name="choose_alarm_time">アラーム時間を選択</string>
+    <string name="reminders">アラーム</string>
+    <string name="usage">注：アラームを設定するには、講義を長押しします。</string>
+    <string name="libraries">" \t\tこのアプリは次のライブラリを使用しています。T\t\\tKotlin, Google Support Library, BetterLinkMovementMethod, Email Intent Builder, \t\tEngelsystem, Espresso, OkHttp, SnackEngage, TraceDroid \t"</string>
+    <string name="about_data_privacy_statement">"         <![CDATA[<a href="https://github.com/EventFahrplan/EventFahrplan/blob/master/DATA-PRIVACY-DE.md">データプライバシーステートメント（ドイツ語）</a>]]>     "</string>
+    <string name="issues">" \t\t<![CDATA[<a href="https://github.com/EventFahrplan/EventFahrplan/issues">問題または機能のリクエスト</a>]]> \t"</string>
+    <string name="today">今日</string>
+    <string name="schedule_parsing_error_generic">スケジュールの解析中に不明なエラーが発生しました</string>
+    <string name="schedule_parsing_error_with_version">スケジュールバージョン％sの解析エラー %s</string>
+    <string name="engelsystem_shifts_parsing_error_generic">Engelsystemシフトの解析中に不明なエラーが発生しました</string>
+    <string name="engelsystem_shifts_parsing_error_forbidden">Engelsystemシフトの無効なAPIキー</string>
+    <string name="engelsystem_shifts_parsing_error_not_found">EngelsystemシフトのURLが間違っています</string>
+    <string name="alarms_empty">表示するアラームはありません。 スケジュールの項目ごとに個別のアラームを設定できることに注意してください。 アクションバーのアラームアイコンを押すだけです。</string>
+    <string name="add_to_calendar_failed">カレンダーへのイベントの追加に失敗しました。</string>
+    <string name="no_changes">スケジュール変更がありません</string>
+    <string name="no_favorites">好きな講義がありません</string>
+    <string name="schedule_changes">スケジュール変更</string>
+    <string name="general_settings">一般</string>
+    <string name="starred_sessions">お気に入り</string>
+    <string name="invalid_url">無効なURL</string>
+    <string name="choose_to_delete">講義を選択</string>
+    <string name="day_separator">日付 <xliff:g example="2" id="day_of_congress">%1$d</xliff:g> - <xliff:g example="29.12.2014" id="date_of_congress_day">%2$s</xliff:g></string>
+    <string name="session_duration"><xliff:g example="45" id="duration">%1$d</xliff:g>分</string>
+    <string name="dlg_delete_all_favorites">すべてのお気に入りを削除しますか？</string>
+    <string name="dlg_delete_all_favorites_delete_all">すべて削除</string>
+    <string name="share_error_activity_not_found">フィッティングアプリがインストールされていません。</string>
+    <string name="build_info_time">"         立ち上げの時間: <xliff:g example="2015-12-27T13:42Z" id="build_time">%s</xliff:g>     "</string>
+    <string name="build_info_version_code">"         バージョンコード: <xliff:g example="42" id="version_code">%s</xliff:g>     "</string>
+    <string name="build_info_hash">"         バージョンハッシュ <xliff:g example="e1f2g3h-dirty" id="build_hash">%s</xliff:g>     "</string>
+    <string name="app_disclaimer">"         このアプリケーションは、&#8220;EventFahrplan&#8221;　アプリのソースコードに基づいています。 Chaos Computer Clubが主催する年次会議やキャンプの訪問者は、このアプリを使用してイベントプログラムを追跡します。"</string>
```

Comment:
Please remove the surrounding quotes and whitespace. It might have been added by the editor.

Reply:
Thanks for removing the quotes. The whitespace can be removed, too.